chart/templates/iq-server-deployment.yaml

```diff
@@ -218,7 +218,14 @@ spec:
       initContainers:
         - name: {{ .Release.Name }}-set-iq-persistence-ownership
           image: busybox:1.28
-          command: ["sh", "-c", "chown -R 1000:1000 {{ .Values.iq_server.config.clusterDirectory }}"]
+          command:
+            - "sh"
```

Comment:
for consistency purposes lets follow the existing syntax like so https://github.com/sonatype/nexus-iq-server-ha/blob/0ba4ddcb6cd0b43dd1162e614a6b5bd02c902347/chart/templates/iq-server-deployment.yaml#L164
```
              command:
                - /bin/sh
                - -c
                - |
                  << helm templating >>
```